docs/_user_guide/best_practices_optimization.md

```diff
@@ -493,50 +538,47 @@ module purge
 module load gurobipy3 ampl knitro
 
 # Activate venv (can either be a global or relative path)
-source /zfs/projects/<your-project>/opt/bin/activate
+source /zfs/projects/<your-project>/opt_combined/bin/activate
 
-# Run python script
-python <your-script.py>
+# Run Python script
+python <my_script.py>
 ```
 
 In this example, we load all three optimization software, but you can omit the ones you don't need.
 
-Submit to test:
+Submit this Slurm script to test:
 
-```bash
-$ sbatch test-opt.slurm
+```title="Terminal Command"
+sbatch opt_test.slurm
 ```
 
 See your job in the queue:
 
-
-```bash
-$ squeue
+```title="Terminal Command"
+squeue
 ```
 
-
 Display the `.out` file once the job is done.
 
-```bash
-$ cat *.out
+```title="Terminal Command"
+cat *.out
 ```
 
-## Running Job Arrays with Slurm
+### Running Job Arrays
 
-A Slurm job array is a way to launch multiple jobs in parallel. One use case is that you want to change input parameters
-to your script (a Python, Julia, or R script). Instead of manually changing the input parameters and rerunning the script
-multiple times, we can do this in with a single job array.
+A Slurm job array is a way to launch multiple jobs in parallel. One use case is when you want to change input parameters in your script (such as a Python, Julia, or R script). Instead of manually changing the input parameters and rerunning the script multiple times, you can achieve this with a single job array.
 
 #### Gurobi Example
-We will work with the following python script that was modified from <a href="https://www.gurobi.com/documentation" target="_blank">Gurobi documentation</a>.
 
-This scripts formulates and solves the following simple MIP model using the Gurobi matrix API:
+We will work with the following Python script that was modified from [Gurobi documentation](https://www.gurobi.com/documentation){:target="_blank"} that performs sensitivity analysis using the Gurobi optimization library.
+
+Specifically, this script formulates and solves a simple Mixed Integer Programming (MIP) model using the Gurobi matrix API:
 
-![](../images/gurobi-eq.png)
+![gurobi equation image](/assets/images/gurobi-eq.png)
 
-Save this python script to a new file called `gurobi_example.py`.
+Save this Python script to a new file named `gurobi_sensitivity.py`.
 
-```python
+```python linenums="1" hl_lines="8-12" title="gurobi_sensitivity.py"
```

Comment:
did you mean to highlight threads?

Reply:
Yes, since this is a good practice to avoid resource contention, and those lines are also listed below. I can remove them.